draft-ietf-quic-transport.md

```diff
@@ -994,6 +994,19 @@ packets sent from only one local address.  An endpoint that migrates away from a
 local address SHOULD retire all connection IDs used on that address once it no
 longer plans to use that address.
 
+An endpoint can request that its peer retire connection IDs by sending a
+NEW_CONNECTION_ID frame with an increased Retire Prior To field.  Upon receipt,
+the peer SHOULD retire the corresponding connection IDs and send the
+corresponding RETIRE_CONNECTION_ID frames in a timely manner.  Failing to do so
+can cause packets to be delayed, lost, or cause the original endpoint to send a
+stateless reset in response to a connection ID it can no longer route correctly.
+
+An endpoint MAY discard a connection ID for which retirement has been requested
+once an interval of no less than 3 PTO has elapsed since an acknowledgement is
+received for the NEW_CONNECTION_ID frame requesting that retirement.  Subsequent
+incoming packets using that connection ID will elicit a response with the
```

Comment:
You want to say "Subsequent incoming packets using that connection ID **MAY** elicit a response with the corresponding stateless reset token."

Note that both behaviors are possible: MAY drop packets silently (misrouted, for example), and/or MAY elicit a stateless reset.

Reply:
If by "MAY" you mean "could", that makes sense.  "MAY" in the sense of permitting something, is not really in question here.

Reply:
Yes, that's right.